src/Mvc/Mvc.Api.Analyzers/test/ActualApiResponseMetadataFactoryTest.cs

```diff
@@ -14,6 +14,12 @@ public class ActualApiResponseMetadataFactoryTest
 {
     private static readonly string Namespace = typeof(ActualApiResponseMetadataFactoryTest).Namespace;
 
+    public enum ReturnOperationTestVariant
```

Comment:
I opted to use an enum to differentiate between the types of test cases, in order to make things a bit clearer, but if this seems overkill I can change this to a boolean instead.